modules/installation-cloudformation-subnet-localzone.adoc

```diff
@@ -1,42 +1,61 @@
 // Module included in the following assemblies:
 //
-// * installing/installing_aws/installing-aws-localzone.adoc
-// * post_installation_configuration/aws-compute-edge-tasks.adoc
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones) 
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with compute nodes on AWS Wavelength Zones) 
+// * post_installation_configuration/aws-compute-edge-zone-tasks.adoc (AWS zone tasks)
 
 :_mod-docs-content-type: REFERENCE
 [id="installation-cloudformation-subnet-localzone_{context}"]
-= CloudFormation template for the subnet that uses AWS Local Zones
+= CloudFormation template for the VPC Subnet
```

Comment:
@mtulio . Is this correcT

Reply:
I do not believe adding Amazon is correct @dfitzmau. Also - I think this would fall under the separate effort of going back to make sure we correctly references Amazon Web Services Virtual Private Cloud (VPC) across our docs.